README.md

```diff
@@ -1 +1,240 @@
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **The user can signup and login to the website to make reservation.**
+- **The user can see links to: Yachts, Reserve, My Reservations, Add Yacht and Delete Yacht**
+- **The user can see a list of yachts**
+- **The user  can select the item and see details of the item**
+- **The user clicks the "Add yacht" link in the navigation panel they can see a form for adding a new yacht.**
+- **When the user clicks the "Delete yacht" link in the navigation panel they can see a list of all items with title and "Delete" button.**
+- **When the user clicks the "My reservations" link in the navigation panel they can see a list of their reservations (with information about item name, date and city).**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- <a href="https://beta-yacht-rental.netlify.app/">Yacht Rental Live</a>
+- <a href="https://beta-yacht-rental.onrender.com/api-docs/index.html">API Documentation</a>
+
+##  ERD Diagram <a name="erd-diagram"></a>
+
+<img src="./image.png" alt="logo" width="auto"  height="auto" />
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Kanban Board <a name="Kanban-Board"></a>
+
+ ## A Team of 3 Members worked on this project - [See list of authors](#authors)
+   - <a href="https://github.com/K0ppai/yacht-rental-backend/projects/1">Kanban Board Link</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- `React`
+- `Redux`
+- `Rails`
+- `Tailwind CSS and HTML`
+- `Code editor (VSCode)`
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  git clone https://github.com/K0ppai/yacht-rental-backend.git
+```
+
+### Install
+
+Install this project run these commands in order:
+
+```sh
+  cd my-project
+  bundle install
+  rails db:create
+  rails db:migrate
+  rails db:seed
+```
+
+### Usage
+
+
+To run the project, execute the following command:
+
+```sh
+  rails server
+```
+
+To run tests, run the following command:
+
+```sh
+  rspec ./spec/models
+```
+To run tests for controllers(request) and for API Documentation, run the following command:
+
+Install the gem -
+ Add the `gem 'rswag'` and `gem 'rswag-ui'` in your gem file and run `bundle install`
+
+Run
+
+```sh
+  rails generate rswag:install
+```
+
+```sh
+  rake rswag:specs:swaggerize
+```
+### If you're trying to run the app on local, kindly set up the following,
+
+- Generate a secret by typing this cmd in the terminal:
+```sh
+ bundle exec rake secret
+``` 
+
+```sh
+ EDITOR='code --wait' rails credentials:edit 
+``` 
```

Comment:
- [ ] In order for this command to work, you need to delete `config/credentials.yml.enc` first.  Kindly, reflect this on your instructions.

Otherwise, it throws this error:

![image](https://github.com/K0ppai/yacht-rental-backend/assets/57421823/ca2b9512-a5e8-4349-b563-dbe906c5cc13)